### PR TITLE
ITM -555 optimized a couple of queries

### PR DESCRIPTION
**Pull request by phile-caci**
Added projections to a couple of queries to reduce their size. The queries should be much smaller now:
/adm-results 123mb -> 1.88mb
**before:**
image

**after:**
image

/programQuestions  153mb (before eval filter and projection) -> 251kbs
**before:**
image

**after:**
image

### CODE REVIEW

**Comment by nextcen-dgemoets** [comment]
How does one test this?  What parts of the dashboard are affected by this?

**Comment by phile-caci** [comment]
> How does one test this? What parts of the dashboard are affected by this?

Hey @nextcen-dgemoets ,

for /adm-results,
open your browser and go into debug mode (right-click->inspect), select the network tab, then nav to https://darpaitm.caci.com/adm-results

On Prod:
image

Then do the same with this branch... should be much smaller

Then go through the above steps, with the home page, which is the programQuestions.

Hope that helps!

**Review by nextcen-dgemoets — APPROVED** [review]
LGTM!

File: dashboard-ui/src/components/AggregateResults/programQuestions.jsx [hunk 1]
```diff
@@ -147,9 +147,15 @@ export default function ProgramQuestions({ allData, kdmaScatter, chartData }) {
             for (const x of data.getAllHistoryByEvalNumber) {
                 if (x.history.length > 0) {
                     const admName = x.history[0].parameters.adm_name;
-                    let scenario = x.history[0].response.id;
+                    //let scenario = x.history[0].response.id
```
Comment (nextcen-dgemoets) [inline dashboard-ui/src/components/AggregateResults/programQuestions.jsx:150]: Maybe remove this comment.

Comment (phile-caci) [reply dashboard-ui/src/components/AggregateResults/programQuestions.jsx]: > Maybe remove this comment.

Done!

**Review by brianpippin — APPROVED** [review]

**Pull request merged**